contracts/SimpleNftMarketplace.sol

```diff
@@ -92,11 +92,17 @@ contract SimpleNftMarketplace is ListingManager, ValidateSignature {
     return _listings[listingId].buyer == address(0) && _listings[listingId].seller != address(0);
   }
 
-  function isBlacklistedUser(address userAddress) external view returns (bool isBlacklisted) {}
+  function isBlacklistedUser(address userAddress) external view returns (bool isBlacklisted) {
+    return (_isBlacklistedUser(userAddress) == true);
```

Comment:
_isBlacklistedUser(userAddress)  already return true or false, no need to add == true